app/src/userpages/components/StreamPage/constants.js

```diff
@@ -1,12 +1,36 @@
 // @flow
 
 export const leftColumn = {
-    xs: 9,
+    xs: 7,
 }
 
 export const rightColumn = {
     xs: {
-        size: 2,
-        offset: 10,
+        size: 4,
+        offset: 1,
+    },
+}
+
+export const fieldTypes = ['number', 'boolean', 'map', 'list', 'string']
+
+export const fieldValidators = {
+    number: (f: string): boolean => !Number.isNaN(parseFloat(f)),
+    string: (f: string): boolean => f instanceof String || typeof f === 'string',
+    boolean: (f: string): boolean => f === 'true' || f === 'false',
+    map: (f: string): boolean => {
+        try {
+            const m = JSON.parse(f)
+            return typeof m === 'object' && m !== null
```

Comment:
`typeof []` gives `object` too, FYI.

Reply:
Ah, I just moved it to this file. I actually see it's not used anywhere so I might as well remove it.